docs/ja/development-contribution/development-guide.md

```diff
@@ -0,0 +1,148 @@
+# KHI開発環境のセットアップ
```

Comment:
We would better to have a link to Japanese/English on both documents for visitors navigated here with direct link to be aware of the existence of translated document.

Like this: 
![image](https://github.com/user-attachments/assets/db7d006f-accf-42f5-88bb-87d136f7c481)